src/OpenTK/Platform/Linux/LinuxDisplayDriver.cs

```diff
@@ -341,17 +306,13 @@ private unsafe static bool QueryDisplay(int fd, ModeConnector* c, out LinuxDispl
 
         private unsafe void AddDisplay(LinuxDisplay display)
         {
-            DisplayResolution[] modes = new DisplayResolution[display.pConnector->count_modes];
-            DisplayResolution current;
-            GetModes(display, modes, out current);
-
-            bool is_primary = AvailableDevices.Count == 0;
-            DisplayDevice device = new DisplayDevice(current, is_primary,
-                modes, GetBounds(current), display);
+            GetModes(display);
+            bool is_primary = _availableDevices.Count == 0;
```

Comment:
Agree with making them all isPrimary or is_primary. We haven't got a consistent style on underscores or camel case so I'll stick to what ever the rest of the file is dong. But yes, will put "is" on the front.